tests/testthat/test-variables.R

```diff
@@ -105,6 +105,11 @@ with_mock_crunch({
         expect_error(digits(ds$birthyr) <- "0.7", "digit specifications should be an integer")
         expect_error(digits(ds$gender) <- 0, "digit specifications can only be set for numeric variables")
     })
+
+    test_that("APIToURL generates correct URL", {
```

Comment:
Passing observation: maybe we have enough of these web-app functions now that they should move to their own test file.